.github/workflows/zfs-qemu.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Generate OS config and CI type
         id: os
         run: |
-          FULL_OS='["almalinux8", "almalinux9", "centos-stream9", "debian11", "debian12", "fedora40", "fedora41", "freebsd13-4r", "freebsd14-0r", "freebsd14-1s", "ubuntu20", "ubuntu22", "ubuntu24"]'
+          FULL_OS='["almalinux8", "almalinux9", "archlinux", "centos-stream9", "debian11", "debian12", "fedora40", "fedora41", "freebsd13-4r", "freebsd14-0r", "freebsd14-1s", "ubuntu20", "ubuntu22", "ubuntu24"]'
```

Comment:
Archlinux is most of the time ahead our linux-max version in the META file.
I don't think it's useful for the OpenZFS repository. You can add `archlinux` within you own Actions for testings an so on.

Reply:
> You can add `archlinux` within you own Actions for testings an so on.

Sounds good to me. Thanks for the review.